### DISCUSSION TITLE
Nested slider navigation problem on Safari

### DISCUSSION THREAD

**Discussion by vktra** [nolimits4web/swiper#4174] [Q&A]
I've nested several sliders and it works great overall, except on Safari where the navigation arrows only appear after a resize of the window or after the first slide has gone through (by drag and drop or using the keyboard). I forced all the css properties of the arrows (swiper-button-prev and swiper-button-next): opacity, display, z-index... But nothing to do, it still doesn't work.
Here it is : https://www.lilyetlea.fr/atelier-de-tissage
Thanks in advance.
Cheers.

**Comment by nolimits4web** [comment]
Try using `height: 100vh`, using `height: 100%` is not a good candidate for nested relative elements

**Reply by vktra** [reply]
Thanks for the advice but unfortunately it doesn't seem to work...

**Reply by vktra** [reply]
Thanks for your reactivity!
